src/microbe_stage/Microbe.cs

```diff
@@ -1485,7 +1485,8 @@ private void ReadyToReproduce()
             // The player doesn't split automatically
             allOrganellesDivided = true;
 
-            OnReproductionStatus?.Invoke(this, true);
+            if (!Dead)
+                OnReproductionStatus?.Invoke(this, true);
```

Comment:
Wouldn't it be better to have `OnReproductionStatus?.Invoke(this, !Dead);`?

Reply:
I considered that, but thought it might be a little unclear.

Reply:
To me the really unclear thing is why a dead cell is allowed to even call ReadyToReproduce at all.